docker-compose.yml

```diff
@@ -1,15 +1,12 @@
-version: '3'
+version: "3"
 
 services:
   postgres:
     container_name: postgres
-    image: postgres:9.6-alpine
-    volumes:
-      - ./db:/var/lib/postgresql
-      - ./db/data:/var/lib/postgresql/data
```

Comment:
I think we still want these volumes (or at least the data volume) so that the data will persist even if we shut the container down. This way we won't have to upload the dataset each time we redeploy. Did you have any problems with this in LightSail?

Reply:
I agree with @skyripley. Also this PR should be requested to be merged into 'develop' branch. 

- 'develop' branch should be the default branch. 
- 'master' branch should be only use for final checkout and deploy to AWS. 

That was the intention ...